tool/Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-CoreDataModules = {git = "https://www.github.com/AfricasVoices/CoreDataModules", editable = true}
+CoreDataModules = {git = "https://www.github.com/AfricasVoices/CoreDataModules",editable = true}
```

Comment:
whitespace change, please revert

Reply:
must have been the editor...I've reverted